src/reference/asciidoc/handler-advice.adoc

```diff
@@ -49,11 +49,12 @@ For chains that produce a reply, every child element can be advised.
 [[advice-classes]]
 ==== Provided Advice Classes
 
-In addition to providing the general mechanism to apply AOP advice classes, Spring Integration provides three standard advice classes:
+In addition to providing the general mechanism to apply AOP advice classes, Spring Integration provides four standard advice classes:
```

Comment:
Well,  let's fix it for something like `these standard advice classes`  to cover all the possible future additions 😄